pkg/agent/config.go

```diff
@@ -30,7 +30,7 @@ type Config struct {
 	OrganizationID string `yaml:"organization_id"`
 	// ClusterID is the cluster that the agent is scanning.
 	ClusterID     string         `yaml:"cluster_id"`
-	DataGatherers []dataGatherer `yaml:"data-gatherers"`
+	DataGatherers []DataGatherer `yaml:"data-gatherers"`
```

Comment:
data-gatherers seems inconsistent with cluster_id and organization_id. Would _ be better?
```suggestion
	DataGatherers []DataGatherer `yaml:"data_gatherers"`
```

Reply:
:+1: We'll see if we can change it in another PR. Changing that now would require a couple of other changes in the platform side also.